src/userscriptvalidators.ts

```diff
@@ -1,4 +1,4 @@
-export const customSettingIsMalformed = function(customUserScriptSetting: UserscriptRenderReadySetting): boolean | string {
+export const customSettingIsMalformed = (customUserScriptSetting: UserscriptRenderReadySetting): boolean | string => {
```

Comment:
can't these be just `export function customSettingIsMalformed() {}`? what's the benefit of exporting a const?
i guess it doesen't really matter that much if the functions aren't using `this`, but i'd rather export normal functions if possible.

Reply:
I don't see the reason either... I did some googling and I couldn't find much about arrow function import/export scope, so I'm not sure if there's weird behavior or not. It shouldn't matter because it only uses its own local variables.
It does save like 5 bytes though...

Reply:
> can't these be just `export function customSettingIsMalformed() {}`? what's the benefit of exporting a const? i guess it doesen't really matter that much if the functions aren't using `this`, but i'd rather export normal functions if possible.

yes, i was just mindlessly fixing eslint errors and didn't notice it at the time. no real difference between arrow functions, normal functions & their const declaration counterparts but ill fix it up to match everything else